ver/us/splat.yaml

```diff
@@ -2485,6 +2527,19 @@ segments:
     subsections:
     - [0x404220, c]
     - [0x404E40, bin]
+  - [0x404F40, ci4,     world/cloud, 32, 32] # FLIP
+  - [0x405140, palette, world/cloud]
+  - [0x405340, ci4,     world/waterblock, 32, 32] # FLIP
+  - [0x405540, palette, world/waterblock]
+  - [0x405740, ci4,     world/yellow_carpet, 32, 32] # FLIP
+  - [0x405940, palette, world/yellow_carpet]
+  - [0x405B40, i4,     world/number_4, 32, 32] # FLIP ci4
+  - [0x405D40, i4,     world/number_3, 32, 32] # FLIP ci4
+  - [0x405F40, i4,     world/number_2, 32, 32] # FLIP ci4
+  - [0x406140, i4,     world/number_1, 32, 32] # FLIP ci4
```

Comment:
what's going on here

Reply:
They are ci4, but don't work as extracted ci4 since they share a palette.

Reply:
interesting, i should add that feature to splat

Reply:
I think these should all be one texture for now

Reply:
I think I agree, but maybe leave the indivudal ones commented? Because my ideal goal would be to extract these separately probably so editing them is a tad nicer. Eventually we can have like a "spritesheet" group or something which stacks images like this together at build time but extracts them indidivudally for nicer viewing/editing